xesmf/tests/test_frontend.py

```diff
@@ -1007,3 +1007,66 @@ def test_densify_polys():
     poly = Polygon([(-80, -40), (80, -40), (80, 40), (-80, 40)])  # Large poly
     with pytest.warns(UserWarning):
         xe.SpatialAverager(ds_in, [poly])
+
+
+def test_regrid_polekind():
```

Comment:
```suggestion
def test_regrid_polekind(tmp_path):
```
I suggest writing the test files in a temporary path (see https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html).

As a bonus, you don't even need to clean them up at the end.